pybossa/jobs.py

```diff
@@ -1522,3 +1538,62 @@ def respond_csv():
         raise
     finally:
         send_mail(mail_dict)
+
+
+# TODO: uncomment, reuse this under future PR
```

Comment:
I'm wondering if the commented-out code for future PRs should just be kept on a separate branch? Just to avoid confusion. *Minor.*